index.js

```diff
@@ -0,0 +1,106 @@
+function ReadElement(selector) {
+  return document.querySelector(selector);
+}
+
+const menu = ReadElement('#menu-icon');
+const nav = ReadElement('.mobile-nav');
+const closeNav = ReadElement('.close-button');
+
+function AddMenu(selected, modalPart, opener) {
+  return selected.addEventListener('click', () => {
+    modalPart.classList.add(opener);
+    modalPart.style.top = '0px';
+  });
+}
```

Comment:
In the mobile design
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/78251927/bd773b57-f6d9-4c97-af87-96a2fe78885b)

The hamburger button in the `About` page is not functioning properly 👀

- Navigating from `Homepage` to `About` page is working fine. 👍 
- However, in the `About` page the hamburger menu is not working.

Error in the console.
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/78251927/a1a7fdf1-4fe8-4beb-a8bc-d3347ae58c4a)


Please kindly fix that 💪